### PR TITLE
Add more documentation

### PR DESCRIPTION
**Pull request by MoritzWeber0**
Add documentation sections for backups, troubleshooting and files.

### CODE REVIEW

File: docs/user/docs/projects/models/backups/setup.md [hunk 1]
```diff
@@ -2,3 +2,44 @@
...
+<!-- prettier-ignore -->
+!!! warning
+    You need to have the Administrator or Project Lead role for the specific
+    project to perform the following steps.
+
+<!-- prettier-ignore -->
+!!! danger
+    Models are not automatically backed up! Please make sure to set up a nightly backup for productive and important content.
+
+We offer the feature to create a backup from any TeamForCapella repository to any Git repository. These can be triggered manually or can be executed every night at around 3pm.
```
Comment (amolenaar) [inline docs/user/docs/projects/models/backups/setup.md:17]:
```suggestion
We offer a feature to create a backup from any TeamForCapella repository to any Git repository. These can be triggered manually or nightly at around 3pm.
```

is it "manually or nightly" or "manually and nightly"?

File: docs/user/docs/projects/models/backups/setup.md [hunk 2]
```diff
@@ -2,3 +2,44 @@
...
+   In our case, there is not registered backup pipeline yet.
+
+    ![Backup pipeline](./backups-overview.png)
+
+1. Click on the `Plus`-icon on the top right and select the corresponding configuration.
+   You have to select a Git model (source) and a T4C model (destination).
+
+    ![Create a backup pipeline](./create-pipeline.png)
+
+    !!! Bug
```
Comment (amolenaar) [inline docs/user/docs/projects/models/backups/setup.md:39]: Is this really a bug? I think it's more like a warning.

The "Include commit history" does only deal with the new commits to the T4C repo or is it working it's way through the whole repo history?

File: docs/user/docs/sessions/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+??? question "My Capella crashed. What should I do?"
+    Capella can crash for many different reasons. Many issues are issues with Eclipse Capella itself, so we try to escalate issues there. Bugs can be reported to the Eclipse Capella team directly in the [Github repository](https://github.com/eclipse/capella/issues).
+
+    Administrators can see the logs of all sessions to identify the issues remotely. However, the session owner can also see the events in the UI. In your session, please follow these steps:
+
+    1. Open the Capella search
+    1. Search for `Event logs` and click on the following entry:
```
Comment (amolenaar) [inline docs/user/docs/sessions/troubleshooting.md:12]: What entry to search for?

File: docs/user/docs/sessions/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    1. One should now see the list of events. One can try to find an entry with a matching timestamp.
+
+??? question "The clipboard doesn't work anymore."
+
+    This is a known bug with our used `RDP`-server `xrdp`. `xrdp` maintains so called `channels`, which are managed by the process `chansrv`. One channel is `cliprdr`, which enables the clipboard feature. However, the `chansrv` doesn't terminate properly on session disconnects and `chansrv` doesn't accept new connections. This leads to the fact that the clipboard doesn't work anymore after a reconnect. The bug is tracked in the [`xrdp` Github repository](https://github.com/neutrinolabs/xrdp/issues/1188) and in [our Github repository](https://github.com/DSD-DBS/capella-dockerimages/issues/67).
```
Comment (amolenaar) [inline docs/user/docs/sessions/troubleshooting.md:17]: :+1:

**Review by amolenaar — CHANGES_REQUESTED** [review]
Really nice update. Some grammatical improvements, but the content looks sound.

File: docs/user/docs/projects/models/backups/setup.md [hunk 5]
```diff
@@ -2,3 +2,44 @@
...
+# Setup a backup for a TeamForCapella model
+
+<!-- prettier-ignore -->
+!!! warning
+    You need to have the Administrator or Project Lead role for the specific
+    project to perform the following steps.
+
+<!-- prettier-ignore -->
+!!! danger
+    Models are not automatically backed up! Please make sure to set up a nightly backup for productive and important content.
```
[the remark this answers is not in the document]
Comment (MoritzWeber0) [reply docs/user/docs/projects/models/backups/setup.md:15]: Do you want to have the `a` removed as well? 

```suggestion
    Models are not automatically backed up! Please make sure to set up nightly backups for important content.
```

File: docs/user/docs/sessions/troubleshooting.md (hunk 3 above)
Comment (MoritzWeber0) [reply docs/user/docs/sessions/troubleshooting.md:12]: Do you mean which log entry? That depends on Capella and the issue type, e.g. a session crash.

**Comment by MoritzWeber0** [comment]
To prevent merge conflicts, I added the suggestions to https://github.com/DSD-DBS/capella-collab-manager/pull/454 directly.

**Pull request merged**